legal/legal/NOTICE.Spring.txt

```diff
@@ -1,5 +1,5 @@
-Spring Framework 4.3.18.RELEASE
-Copyright (c) 2002-2018 Pivotal, Inc.
+Spring Framework 5.1.8.RELEASE
+Copyright (c) 2002-2019 Pivotal, Inc.
```

Comment:
did you just change the version in the existing file or did you replace the existing file with the corresponding file from the 5.1.8 artifacts?

Reply:
I followed the same as for the spring 4.3.12.RELEASE update. I changed the version and the year in the above file. Also I compared it with the _notice.txt_ from the `spring-core-5.1.8.RELEASE.jar` artifact.

Reply:
My point is, we cannot just update the year but indeed need to replace the existing file with the one from the 5.1.8.RELEASE jar (if it differs in anything else than just the year).

Reply:
There is no difference from the file in 5.1.8.RELEASE jar except the version and the date.